src/components/TenantSelector/stories/basic.story.mdx

```diff
@@ -171,6 +172,26 @@ const validateTenantFailed = async (tenant: string) => {
   </Story>
 </Preview>
 
+### Custom input behavior options
+
+```typescript
+const inputBehaviorOptions = {
+  hasFeedback: false,
```

Comment:
could you use `extra` prop here as well? so it's easier to see what `inputBehaviorOptions` can also do

Reply:
I don't think there is a use for `extra` prop. I just copied it directly from above. Will remove it

Reply:
No, we haven't used it, but it was like an improvement to input configuration. It comes from [antd](https://ant.design/components/form/?locale=en-US#Form.Item)

Reply:
I see! I understand now